lib/_fields.js

```diff
@@ -137,7 +137,7 @@ var fieldDefs = {
   url: {
     type: String,
     regEx: SimpleSchema.RegEx.Url,
-    max: 200
+    max: 2000
```

Comment:
My only ask is that you leave that stack-overflow link in a comment somewhere near here.

Reply:
Good call.